store/sagas/contacts.sagas.js

```diff
@@ -22,141 +21,181 @@ export function* getAll({ domain, token }) {
     },
   });
 
-  // handle response
   try {
-    // TODO: will this block and cause responses to be missed?
-    // use channel instead
     const res = yield take(actions.CONTACTS_GETALL_RESPONSE);
     if (res) {
       const response = res.payload;
       const jsonData = yield response.json();
       if (response.status === 200) {
-        if (jsonData.contacts) {
+        if (jsonData.posts) {
           yield put({
             type: actions.CONTACTS_GETALL_SUCCESS,
-            contacts: jsonData.contacts.map(contact => ({
-              key: contact.ID.toString(),
-              assigned_to: contact.assigned_to.name,
-              groups: contact.groups,
-              is_team_contact: contact.is_team_contact,
-              last_modified: contact.last_modified,
-              locations: [], // contact.locations,
-              milestone_baptized: contact.milestone_baptized,
-              overall_status: contact.overall_status,
-              permalink: contact.permalink,
-              contact_phone: contact.phone_numbers,
-              name: contact.post_title,
-              requires_update: contact.requires_update,
-              seeker_path: contact.seeker_path,
-              shared_with_user: contact.shared_with_user,
-            })),
+            contacts: jsonData.posts,
           });
         } else {
-          // TODO: empty list; display placeholder
+          yield put({
+            type: actions.CONTACTS_GETALL_SUCCESS,
+            contacts: [],
+          });
         }
       } else {
         yield put({
           type: actions.CONTACTS_GETALL_FAILURE,
           error: {
             code: jsonData.code,
             message: jsonData.message,
-            data: jsonData.data,
           },
         });
       }
     }
   } catch (error) {
-    yield put({ type: actions.CONTACTS_GETALL_FAILURE, error: { code: '400', message: '(400) Unable to process the request. Please try again later.' } });
+    yield put({
+      type: actions.CONTACTS_GETALL_FAILURE,
+      error: {
+        code: '400',
+        message: '(400) Unable to process the request. Please try again later.',
+      },
+    });
   }
 }
 
-function* delayPostInitialComment(user, comment, contactId) {
+/* function* delayPostInitialComment(user, comment, contactId) {
```

Comment:
If this feature is being removed temporarily, please add a comment as to why and when it can be added back in.

Reply:
This feature were added back to project.